### ISSUE TITLE
Fatal error: Arguments to path.join must be strings

### ISSUE THREAD

**Issue by mhulse** [yatskevich/grunt-bower-task#122]
Me:

``` bash
# Grunt:
grunt-cli v0.1.13
grunt v0.4.5
# Node:
v0.10.29
# NPM:
1.5.0-alpha-1
```

Problem:

``` bash
$ grunt bower
Running "bower:install" (bower) task
>> Cleaned target dir /.../source/files/plugins
bower not-cached https://github.com/WickyNilliams/headroom.js.git#*
bower resolve https://github.com/WickyNilliams/headroom.js.git#*
bower checkout v0.6.0
Fatal error: Arguments to path.join must be strings
```

I've tried both the Headroom `git` url and straight version number:

``` json
{
  "name": "simple-bower",
  "version": "0.0.0",
  "dependencies": {
    "headroom.js": "https://github.com/WickyNilliams/headroom.js.git"
  }
}
```

A quick Google found this issue:

https://github.com/sapegin/grunt-bower-concat/issues/21

> Component could be either filename or folder, we need folder.

Not sure what that means though.

Anyone else having a problem with the Headroom repo? The Headroom Bower file doesn't look abnormal to me. Should I report the problem via the Headroom repo instead? Is there something they could do to make their `bower.json` more compatible with `grunt-bower-task`?

Thanks!

**Comment by johnkchiu** [comment]
I think the problem is related to this https://github.com/bower/bower/pull/1403 which is fixed in v1.3.8.  Problem is that grunt-bower-task is using ~1.2.0 so it won't install the new version.

**Comment by mhulse** [comment]
Ah, thanks for the clarfication @johnkchiu, I really appreciate it. :+1: 

Looking like #116 then, is the way to go here. I'll close this one and make a comment in #116 with a "please update to v1.3.8" request. :octocat:
